utils/urlencode.go

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2015 Spinpunch, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+package utils
+
+import (
+	"net/url"
+	"strings"
+)
+
+func UrlEncode(str string) string {
```

Comment:
might look at the diffs with stuff in html and html/templatepkgs (0/5)

func HTMLEscapeString(s string) string
func EscapeString(s string) string